R/execution.r

```diff
@@ -126,7 +147,12 @@ execute = function(request) {
     err <<- list()
     
     handle_error <- function(e) {
-        err <<- list(ename = 'ERROR', evalue = toString(e), traceback = list(toString(e)))
+        calls <- head(sys.calls()[-seq_len(23)], -3)
```

Comment:
no idea. i basically let the whole stack trace print and counted where the useful stuff began.

please play around with all that as well, it’s really complex. what i did for debugging:

```r
calls <- NULL
withCallingHandlers(some_failing_expression(), error = function(e) { calls <<- sys.calls() })
```

the i inspected `calls` using `str` and found that calls in user code have a `attr(call, 'srcref')`.